applications/tari_base_node/Cargo.toml

```diff
@@ -38,7 +38,7 @@ futures = { version = "^0.3.16", default-features = false, features = ["alloc"]
 qrcode = { version = "0.12" }
 log = { version = "0.4.8", features = ["std"] }
 log-mdc = "0.1.0"
-log4rs = { version = "1.2.0", default_features = false, features = ["config_parsing", "threshold_filter", "yaml_format", "console_appender", "rolling_file_appender", "compound_policy", "size_trigger", "fixed_window_roller"] }
+log4rs = { git = "https://github.com/tari-project/log4rs.git", default_features = false, features = ["config_parsing", "threshold_filter", "yaml_format", "console_appender", "rolling_file_appender", "compound_policy", "size_trigger", "fixed_window_roller"] }
```

Comment:
This doesn't appear to freeze at any particular version. Does that introduce risk on any breaking changes to the forked repository?

Reply:
See other instances of this elsewhere in the PR.

Reply:
I'm not so worried, as the original repo is essentially dead, and we don't plan on making additional fixes so likely this will stay as is, until we go back to the original if/when they ever update.

Reply:
I mean is there a particular reason to use the main branch for this, instead of including a specific commit reference or tag?